sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleService.java

```diff
@@ -884,23 +884,37 @@ public Predicate buildCriteriaFilter(SampleCriteria criteria, SampleQueryContext
 			}
 		}
 
+		filter = getCaseContactEventParticipantSamplePredicate(criteria, cb, sample, filter);
+
+		return filter;
+	}
+
+	private Predicate getCaseContactEventParticipantSamplePredicate(
```

Comment:
This method does not only create a new Predicate but also add's it on the given filter predicate.

Please rename to `addCaseContactEventParticipantSamplePredicate`